ebmbot/dispatcher.py

```diff
@@ -100,7 +100,10 @@ def run_command(self):
                     cwd=self.cwd,
                     stdout=stdout,
                     stderr=stderr,
-                    env={"EBMBOT_CALLBACK_URL": self.callback_url},
+                    env={
+                        "EBMBOT_CALLBACK_URL": self.callback_url,
+                        "PATH": settings.EBMBOT_PATH or os.environ["PATH"],
```

Comment:
How would `os.environ["PATH"]` be set?

Reply:
I read this as saying: use the default PATH from the environment if we're not specifically overriding it in settings.

Is this a specific question about how PATH is configured in systemd unit files?

Reply:
I don't think POSIX standard *requires* `PATH` to be set and I presume @inglesp means "if it's not set this will break".

My assumption writing this was that it is always set in practice and we'd deal with it if it wasn't

Reply:
Oh I see. Well none of our jobs use absolute paths for their executables so it's already a problem for us if it's not set.